modules/search/instant-search/components/search-box.scss

```diff
@@ -9,32 +10,41 @@ input.jetpack-instant-search__box-input.search-field,
 	background: #fff;
```

Comment:
We might want to consider adding something to override this styling set by the 2020 theme:

```css
input[type="search"]:focus {
    outline: thin dotted;
    outline-offset: -4px;
}
```

`outline: none` applied to `input[type="search"].jetpack-instant-search__box-input` is a viable solution for this.

Reply:
It's probably more appropriate to tackle this in a future PR referenced here: https://github.com/Automattic/jetpack/pull/14664#issuecomment-586216217.